server/websocket_connector.go

```diff
@@ -92,22 +100,49 @@ func (srv *WSHandler) sendLoop() {
 	for {
 		select {
 		case raw := <-srv.sendChannel:
-			if guble.DebugEnabled() {
-				if len(raw) < 80 {
-					guble.Debug("send to client (userId=%v, applicationId=%v, totalSize=%v): %v", srv.userId, srv.applicationId, len(raw), string(raw))
-				} else {
-					guble.Debug("send to client (userId=%v, applicationId=%v, totalSize=%v): %v...", srv.userId, srv.applicationId, len(raw), string(raw[0:79]))
+
+		   if(srv.checkAccess(raw)) {
+				if guble.DebugEnabled() {
+					if len(raw) < 80 {
+						guble.Debug("send to client (userId=%v, applicationId=%v, totalSize=%v): %v", srv.userId, srv.applicationId, len(raw), string(raw))
+					} else {
+						guble.Debug("send to client (userId=%v, applicationId=%v, totalSize=%v): %v...", srv.userId, srv.applicationId, len(raw), string(raw[0:79]))
+					}
+				}
+
+				if err := srv.clientConn.Send(raw); err != nil {
+					guble.Info("applicationId=%v closed the connection", srv.applicationId)
+					srv.cleanAndClose()
+					break
 				}
 			}
-			if err := srv.clientConn.Send(raw); err != nil {
-				guble.Info("applicationId=%v closed the connection", srv.applicationId)
-				srv.cleanAndClose()
-				break
-			}
 		}
 	}
 }
 
+func (srv *WSHandler) checkAccess(raw []byte) bool {
+	guble.Debug("raw message: %v", string(raw))
+	if raw[0] == byte('/') {
+		path := getPathFromRawMessage(raw)
+		guble.Debug("Received msg %v %v", srv.userId, path)
+		return len(path) == 0 || srv.accessManager.AccessAllowed(READ, srv.userId, path)
+
+	}
+	return true;
+}
+
+func getPathFromRawMessage(raw []byte) guble.Path {
```

Comment:
this function should be in 'guble/message.go'
Maybe it is simpler to get the path by something like:
i := strings.Index(string(raw), ",")
return guble.Path(raw[:i])